lib/ecto/exceptions.ex

```diff
@@ -204,13 +204,17 @@ defmodule Ecto.MultipleResultsError do
   defexception [:message]
 
   def exception(opts) do
-    query = Keyword.fetch!(opts, :queryable) |> Ecto.Queryable.to_query
+    sql_query = Keyword.get(opts, :sql_query)
```

Comment:
There is a case when we don't have Ecto.query (it comes in `struct/7` for delete statements).